### PR TITLE
Allow to clone repositories with None product_id

### PR DESCRIPTION
**Pull request by bliuredhat**
JIRA:PDC-1482

### CODE REVIEW

File: pdc/apps/repository/tests.py [hunk 1]
```diff
@@ -498,6 +498,13 @@ def test_skip_on_include_product_id(self):
         self.assertItemsEqual(response.data, [self.repo2])
         self.assertNumChanges([1])
 
+    def test_with_product_id_value_null(self):
+        args = {'release_id_from': 'release-1.1', 'release_id_to': 'release-1.0',
+                'include_product_id': None}
```
Comment (lubomir) [inline pdc/apps/repository/tests.py]: Should `include_product_id` have some effect on the result here?

Comment (bliuredhat) [inline pdc/apps/repository/tests.py]: @lubomir , I will add .

**Comment by lubomir** [comment]
:+1:

**Comment by simozhan** [comment]
:+1:

**Pull request merged**